### PR TITLE
Require dev16 for integration testing

### PR DESCRIPTION
**Pull request by sharwell**
The vs-deps branches have dependencies on Visual Studio 2019 that cannot be fulfilled on Visual Studio 2017. This change updates these branches to require Visual Studio 2019 for integration testing.

### CODE REVIEW

**Comment by jaredpar** [comment]
Do we have the Vs2019P2 images ready yet for use here? Or are just short circuiting the integration tests?

**Comment by sharwell** [comment]
@jaredpar Just short-circuit the integration tests. They fail in 15 minutes instead of 2hrs 15mins.

**Comment by jaredpar** [comment]
Okay. I will ping the thread again about getting the VS2019P2 images.

**Comment by jaredpar** [comment]
Actually given this is the preview3-vs-deps branch why not just yank the branch from the pipeline? It won't affect anything else and this branch will never RI. Gets the failure time to 0.

**Comment by sharwell** [comment]
> Actually given this is the preview3-vs-deps branch why not just yank the branch from the pipeline? It won't affect anything else and this branch will never RI. Gets the failure time to 0.

We could, but we still want this dependency to merge forward to all the other vs-deps branches.

File: azure-pipelines-integration.yml [hunk 1]
```diff
@@ -3,9 +3,7 @@ trigger:
 - master
 - master-vs-deps
 - dev16.0-preview2
-- dev16.0-preview2-vs-deps
 - dev16.0-preview3
-- dev16.0-preview3-vs-deps
```
Comment (jasonmalinowski) [inline azure-pipelines-integration.yml]: Why isn't this deleting the other -vs-deps, because it'll break those too, right?

Comment (sharwell) [reply azure-pipelines-integration.yml]: master-vs-deps and newer are expected to work with integration tests running against 2019

**Pull request merged**